frontend/components/Domain/Recipe/RecipePage/RecipePageParts/RecipePageInstructions.vue

```diff
@@ -55,6 +55,9 @@
               {{ $t("recipe.auto") }}
             </BaseButton>
             <BaseButton class="ml-2" save @click="setIngredientIds"> </BaseButton>
+            <BaseButton v-if="availableNextStep" class="ml-2" @click="saveAndOpenNextLinkIngredients">
+              <template #icon> {{ $globals.icons.save }}</template>
```

Comment:
i would suggest to use another button. 
maybe the foreward button would be a good fit

![image](https://github.com/mealie-recipes/mealie/assets/24235032/80923010-53c7-4bc9-80a6-2dfc916b49b8)


```suggestion
              <template #icon> {{ $globals.icons.forward }}</template>
```

Reply:
Initially wanted to make clear the button also saves, but I guess the green color already kinda points to that.
The double usage of the same icon is definitely weird and ugly, so good suggestion 👍